elftools/elf/elffile.py

```diff
@@ -136,6 +136,7 @@ def address_offsets(self, start, size=1):
         """
         end = start + size
         for seg in self.iter_segments():
+            if (seg['p_type'] != 'PT_LOAD'): continue
```

Comment:
Please move `continue` to the next line; parens around condition aren't necessary; and also add a comment explaining this condition

Reply:
Commited :)